ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -1133,6 +1133,7 @@ device_classes:
                     disk:
                         type: COMMAND
                         commandTemplate: |-
+                            $$ZENOTHING
```

Comment:
What's $$ZENOTHING do?

Reply:
It's to suppress zCommandPath being prepended to the command. RRDDataSource will prefix the command with zCommandPath unless the command begins with a $ or /.

https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/ZenModel/RRDDataSource.py#L245-L258